fml/backtrace.cc

```diff
@@ -74,12 +88,15 @@ std::string BacktraceHere(size_t offset) {
 static size_t kKnownSignalHandlers[] = {
     SIGABRT,  // abort program
     SIGFPE,   // floating-point exception
-    SIGBUS,   // bus error
+    SIGTERM,  // software termination signal
     SIGSEGV,  // segmentation violation
+#if !FML_OS_WIN
+    SIGBUS,   // bus error
     SIGSYS,   // non-existent system call invoked
     SIGPIPE,  // write on a pipe with no reader
     SIGALRM,  // real-time timer expired
-    SIGTERM,  // software termination signal
+
```

Comment:
```suggestion
```

Reply:
done

Reply:
Without this line, the formatter makes additional whitespace to comment below to align.
```c++
    SIGALRM,  // real-time timer expired
#endif        // !FML_OS_WIN
// Here ^^^
```
Should we keep this line to remove these space, or remove this line?